src/test/specs/GetTestDataFileSpec.ts

```diff
@@ -0,0 +1,17 @@
+import { describeCommon } from '../TestHelper';
+import { TestUtils } from '../..';
+import { assert } from 'chai';
+
+export const YOUR_ATTRIBUTE = process.env.YOUR_ATTRIBUTE!;
+
+/**
+ * GetTestDataFileSpec
+ */
+
+describeCommon('GetTestDataFileSpec', () => {
+  it('Check get test data from env file', () => {
```

Comment:
please add more spec for example to cover     return data[dataTag] || data;

Reply:
added